packages/manager/modules/pci/src/projects/project/storages/databases/database/advanced-configuration/advanced-configuration.html

```diff
@@ -0,0 +1,155 @@
+                                            />
+
+                                            <oui-radio-group
+                                                name="field-{{$index}}"
+                                                ng-if="$ctrl.model[$index].key.type === $ctrl.FIELD_TYPES.boolean"
+                                                model="$ctrl.model[$index].value"
+                                                inline
+                                            >
+                                                <oui-radio
+                                                    disabled="$ctrl.pending"
+                                                    inline
+                                                    value="true"
+                                                    >True</oui-radio
+                                                >
+                                                <oui-radio
+                                                    disabled="$ctrl.pending"
+                                                    inline
+                                                    value="false"
+                                                    >False</oui-radio
+                                                >
+                                            </oui-radio-group>
+
+                                            <oui-select
+                                                disabled="$ctrl.pending"
+                                                name="field-{{$index}}"
+                                                ng-if="$ctrl.model[$index].key.values"
+                                                data-model="$ctrl.model[$index].value"
+                                                data-items="$ctrl.model[$index].key.values"
+                                                class="oui-select__input"
+                                            >
+                                            </oui-select>
+                                        </oui-field>
+                                    </div>
+                                </div>
+                            </fieldset>
+                            <footer
+                                class="oui-inline-adder__footer"
+                                ng-if="!$ctrl.model[$index].isFixed"
+                            >
+                                <button
+                                    type="button"
+                                    class="oui-inline-adder__action oui-button oui-button_primary oui-button_s"
+                                    data-ng-click="$ctrl.onAddConfig($index)"
+                                    data-ng-if="!$ctrl.model[$index].added"
+                                    data-ng-disabled="!$ctrl.model[$index].key || $ctrl.addableProperties.length <= 1 || $ctrl.pending"
+                                >
+                                    <span class="oui-icon oui-icon-add"></span>
```

Comment:
We should labels the button element.
```suggestion
                                    <span class="oui-icon oui-icon-add" aria-hidden="true"></span>
```

Reply:
Added here: https://github.com/ovh/manager/pull/6568/commits/d41de0c46bc372d3834ffe66082fcb7eca3aa01d